src/server/pegasus_server_impl_init.cpp

```diff
@@ -14,6 +15,18 @@
 
 namespace pegasus {
 namespace server {
+
+DSN_DEFINE_int64(
+    "pegasus.server",
+    rocksdb_limiter_max_write_megabytes_per_sec,
+    500,
+    "max rate of rocksdb flush and compaction(MB/s), if less than or equal to 0 means close limit");
+
+DSN_DEFINE_bool("pegasus.server",
+                rocksdb_limiter_auto_tune_enable,
+                false,
+                "whether to enable write rate auto tune when open rocksdb write limit");
```

Comment:
How about add those two config in `config.ini` and `config.min.ini`?

Reply:
`config.ini` need, `config.min.ini` can be ignored